app/helpers/submissions_helper.rb

```diff
@@ -1,6 +1,6 @@
 module SubmissionsHelper
   def can_review?
```

Comment:
Is this helper necessary? I _think_ you could use the callback style of writing the ability and do the `submission.section.present?` logic inside the block.